tests/Constraints/VeryBaseTestCase.php

```diff
@@ -87,4 +87,22 @@ private function getJsonSchemaDraft04()
 
         return $this->jsonSchemaDraft04;
     }
+
+    public function expectExceptionCompat($exception)
+    {
+        if (\is_callable(array('PHPUnit\Framework\TestCase', 'expectException'))) {
+            parent::expectException($exception);
+        } else {
+            $this->setExpectedException($exception);
+        }
+    }
+
+    public static function assertIsArrayCompat($actual, $message = '')
+    {
+        if (\is_callable(array('PHPUnit\Framework\TestCase', 'assertIsArray'))) {
+            parent::assertIsArray($actual, $message);
+        } else {
+            static::assertInternalType('array', $actual, $message);
+        }
+    }
```

Comment:
Target PHP versions do not support traits, therefore we can't use [off-the-shelf polyfill](https://github.com/PHPUnitGoodPractices/polyfill).

Reply:
what about replacing `assertInternalType` with an `assertTrue(is_array($data), $message)` instead?

Reply:
Following the same logic we should use `assertTrue(is_array())` instead of `assertIsArray()`; what do you think?